internal/namespaces/instance/v1/custom_image.go

```diff
@@ -35,24 +35,25 @@ func imageCreateBuilder(c *core.Command) *core.Command {
 
 	c.ArgsType = reflect.TypeOf(customCreateImageRequest{})
 
-	originalRun := c.Run
-
-	c.Run = func(ctx context.Context, argsI interface{}) (i interface{}, e error) {
-		args := argsI.(*customCreateImageRequest)
-
-		request := args.CreateImageRequest
-		request.RootVolume = args.SnapshotID
-		request.ExtraVolumes = make(map[string]*instance.VolumeTemplate)
-
-		// Extra volumes need to start at volumeIndex 1.
-		volumeIndex := 1
-		for _, volume := range args.AdditionalVolumes {
-			request.ExtraVolumes[strconv.Itoa(volumeIndex)] = volume
-			volumeIndex++
-		}
+	c.Interceptor = core.CombineInterceptor(
```

Comment:
We could also do somethinf like 
```
c.AddInterceptor(func(ctx context.Context, argsI interface{}, runner core.CommandRunner) (i interface{}, err error) {

})
```

Reply:
Yep I don't like the `core.CombineInterceptor(c.Interceptor,` too